samples/bootloader/README.rst

```diff
@@ -0,0 +1,170 @@
+.. _bootloader:
+
+Immutable bootloader
```

Comment:
Is it correct that this is the immutable bootloader?

Reply:
Yes, this is the immutable bootloader.